src/SFA.DAS.Authorization.WebApi/HttpFilterCollectionExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿#if NET462
+using System.Web.Http.Filters;
+
+namespace SFA.DAS.Authorization.WebApi
+{
+    public static class HttpFilterCollectionExtensions
+    {
+        public static void AddAuthorizationFilter(this HttpFilterCollection filters)
+        {
+            filters.Add(new AuthorizationFilter());
```

Comment:
Do you think it worthwhile to introduce an interface and resolve through `DependencyResolver`?

Reply:
Same as https://github.com/SkillsFundingAgency/das-authorization/pull/2#discussion_r225225036